tests/arista_eos/show_interfaces_transceiver_hardware/arista_eos_show_interfaces_transceiver_hardware_01.yml

```
@@ -0,0 +1,34 @@
+---
+<<<<<<< HEAD
+parsed_sample: []
+=======
+parsed_sample:
+  - cable_type: ""
+    max_module_power_w: "0.0"
+    max_slot_power_w: "N/A"
+    media_type: "1000BASE-T"
+    module_presence: ""
+    name: "Et1"
+    wavelength_nm: ""
+  - cable_type: ""
+    max_module_power_w: "1.0"
+    max_slot_power_w: "N/A"
+    media_type: "10GBASE-CR"
+    module_presence: ""
+    name: "Et49"
+    wavelength_nm: ""
+  - cable_type: ""
+    max_module_power_w: "1.0"
+    max_slot_power_w: "N/A"
+    media_type: "10GBASE-CR"
+    module_presence: ""
+    name: "Et50"
+    wavelength_nm: ""
+  - cable_type: ""
+    max_module_power_w: "1.0"
+    max_slot_power_w: "N/A"
+    media_type: "1000BASE-LX"
+    module_presence: ""
+    name: "Et51"
+    wavelength_nm: "1310"
+>>>>>>> b5072072133c233bbf4f55be40d5efb31cc48180
```

Remove git conflict markers.

```suggestion
---
parsed_sample:
  - cable_type: ""
    max_module_power_w: "0.0"
    max_slot_power_w: "N/A"
    media_type: "1000BASE-T"
    module_presence: ""
    name: "Et1"
    wavelength_nm: ""
  - cable_type: ""
    max_module_power_w: "1.0"
    max_slot_power_w: "N/A"
    media_type: "10GBASE-CR"
    module_presence: ""
    name: "Et49"
    wavelength_nm: ""
  - cable_type: ""
    max_module_power_w: "1.0"
    max_slot_power_w: "N/A"
    media_type: "10GBASE-CR"
    module_presence: ""
    name: "Et50"
    wavelength_nm: ""
  - cable_type: ""
    max_module_power_w: "1.0"
    max_slot_power_w: "N/A"
    media_type: "1000BASE-LX"
    module_presence: ""
    name: "Et51"
    wavelength_nm: "1310"
```